spec/acceptance/php_spec.rb

```diff
@@ -10,12 +10,10 @@
     end
 
     case default[:platform]
-    when %r{ubuntu-18.04}
```

Comment:
I'm generally a fan of smaller diffs and keeping it newest to oldest is OK IMHO. It's still sorted and consistent.